src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,61 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_CAPACITY = 10;
+
+    private Object[] keys = new Object[MAX_CAPACITY];
+    private Object[] values = new Object[MAX_CAPACITY];
```

Comment:
use the parameterized types and constructor to initialize them
```suggestion
    private K [] keys;
    private V [] values;
```